Dockerfile

```diff
@@ -77,17 +77,110 @@ RUN echo $'[carepo]\n\
+     yum install -y \
+     ca_DigiCertAssuredIDRootCA-Root \
+     ca_USERTrustRSACertificationAuthority \
+     ca_DigiCertGridRootCA-Root \
+     ca_CERN-Root-2 \
+     ca_CNRS2 \
+     ca_CNRS2-Projets \
+     ca_QuoVadis-Root-CA2G3 \
+     ca_COMODO-RSA-CA \
+     ca_CESNET-CA-Root \
+     ca_UKeScienceRoot-2007 \
+     ca_QuoVadis-Root-CA3G3 \
+     ca_DarkMatterPrivateRootCAG4 \
+     ca_KENETROOTCA \
+     ca_KENETCA-ICA-2015 \
+     ca_DarkMatterIGTFCA \
+     ca_DarkMatterAssuredCA \
+     ca_UKeScienceCA-2B \
+     ca_CESNET-CA-4 \
+     ca_InCommon-IGTF-Server-CA \
+     ca_DarkMatterSecureCA \
+     ca_CNRS2-Grid-FR \
+     ca_CERN-GridCA \
+     ca_DigiCertGridCA-1G2-Classic-2015 \
+     ca_DigiCertGridCA-1-Classic \
+     ca_GEANTeScienceSSLCA4 \
+     ca_GEANTeSciencePersonalCA4 \
+     ca_DigiCertGridTrustCA-Classic \
+     ca_DigiCertGridTrustCAG2-Classic \
+     ca_TERENAeSciencePersonalCA3 \
+     ca_TERENA-eScience-SSL-CA-3 \
+     ca_IHEP-2013 \
+     ca_PKIUNAMgrid \
+     ca_HKU-CA-2 \
+     ca_GridCanada \
+     ca_ANSPGrid \
+     ca_CNIC \
+     ca_UNLPGrid \
+     ca_NIKHEF \
+     ca_MARGI \
+     ca_NCSA-slcs-2013 \
+     ca_INFN-CA-2015 \
+     ca_KEK \
+     ca_DCAROOT-G1 \
+     ca_IGCA2 \
+     ca_HPCI \
+     ca_KISTIv3 \
+     ca_seegrid-ca-2013 \
+     ca_MaGrid \
+     ca_BYGCA \
+     ca_NorduGrid-2015 \
+     ca_PolishGrid \
+     ca_PolishGrid-2019 \
+     ca_AC-GRID-FR-Services \
+     ca_QuoVadis-Root-CA1 \
+     ca_BG-ACAD-CA \
+     ca_AC-GRID-FR \
+     ca_MYIFAM \
+     ca_TRGrid \
+     ca_REUNA-ca \
+     ca_RDIG \
+     ca_UGRID-G2 \
+     ca_cilogon-silver \
+     ca_MD-Grid-CA-T \
+     ca_QuoVadis-Root-CA2 \
+     ca_SlovakGrid \
+     ca_QuoVadis-Grid-ICA-G2 \
+     ca_SiGNET-CA \
+     ca_MREN-CA \
+     ca_SDG-G2 \
+     ca_PSC-Myproxy-CA \
+     ca_UNAMgrid-ca \
+     ca_ArmeSFo \
+     ca_GermanGrid \
+     ca_LIPCA \
+     ca_HellasGrid-CA-2016 \
+     ca_PK-Grid-2007 \
+     ca_DFN-GridGermany-Root \
+     ca_AC-GRID-FR-Robots \
+     ca_AC-GRID-FR-Personnels \
+     ca_NIIF-Root-CA-2 \
+     ca_RomanianGRID \
+     ca_IRAN-GRID-GCG-G2 \
+     ca_SRCE \
+     ca_ASGCCA-2007 \
+     ca_NERSC-SLCS \
+     ca_AEGIS \
+     ca_TSU-GE \
+     ca_DZeScience \
+     ca-policy-egi-core
+
+# Install VOMS
+RUN yum install -y epel-release && \
+    yum install -y voms-clients-java voms-clients-cpp  fetch-crl globus-gsi-sysconfig
+
+ADD etc/vomses /etc/vomses
 
 # Create truststore for NXCALS Spark connection
 RUN yum -y install java-1.8.0-openjdk && \
     keytool -import -alias cerngridCA -file /etc/pki/tls/certs/CERN_Grid_Certification_Authority.crt \
         -keystore /etc/pki/tls/certs/truststore.jks -storepass 'password' -noprompt && \
     keytool -import -alias cernRootCA2 -file /etc/pki/tls/certs/CERN_Root_Certification_Authority_2.crt \
-        -keystore /etc/pki/tls/certs/truststore.jks -storepass 'password' -noprompt && \
-    yum -y erase java-1.8.0-openjdk && \
```

Comment:
So now we need java inside the container for this voms client right? It's getting really bloated

Reply:
Java is already in the LCG releases, so we should take it from there instead of installing it in the image:

```shell
[etejedor@lxplus737 ~]$ which java
/cvmfs/sft.cern.ch/lcg/views/LCG_97a/x86_64-centos7-gcc8-opt/bin/java
[etejedor@lxplus737 ~]$ java -version
openjdk version "1.8.0_222"
OpenJDK Runtime Environment (build 1.8.0_222-b10)
OpenJDK 64-Bit Server VM (build 25.222-b10, mixed mode)
```